gradle.properties

```diff
@@ -1,7 +1,7 @@
 # Main properties
 group=com.truelayer
 archivesBaseName=truelayer-java
-version=8.0.0
+version=9.0.0
```

Comment:
I think this is a breaking change given we had to rename / remove properties on which some clients might have applied their logic